### PR TITLE
fix(vite): inform user to add build config

### PR DESCRIPTION
**Pull request by mandarini**
If a `vite.config.ts` file already exists when we are converting an existing project to use vite, make sure to edit that file to add the build settings, or modify the existing build settings.

### CODE REVIEW

**Comment by jaysoo** [comment]
Looks good so far.

File: packages/vite/src/migrations/update-15-3-1/update-vite-tsconfig-paths.ts [hunk 1]
```diff
@@ -7,6 +7,7 @@ import {
 } from '@nrwl/devkit';
 
 import { forEachExecutorOptions } from '@nrwl/workspace/src/utilities/executor-options-utils';
+import { tsquery } from '@phenomnomnominal/tsquery';
```
Comment (jaysoo) [inline packages/vite/src/migrations/update-15-3-1/update-vite-tsconfig-paths.ts:10]: Should be `15-4-0` in the file path (minor).

Also needs to be added to `migrations.json` file.

File: packages/vite/src/utils/generator-utils.ts [hunk 2]
```diff
@@ -696,3 +735,230 @@ export async function handleUnknownExecutors() {
     `);
   }
 }
+
+export function checkForBuildOptionsInViteConfig(
```
Comment (jaysoo) [inline packages/vite/src/utils/generator-utils.ts:739]: Can we name this `ensureBuildOptionsInViteConfig`? "check" is implying it doesn't change anything.

**Review by jaysoo — APPROVED** [review]

File: packages/vite/src/migrations/update-15-3-1/update-vite-tsconfig-paths.ts (hunk 1 above)
Comment (mandarini) [reply packages/vite/src/migrations/update-15-3-1/update-vite-tsconfig-paths.ts:10]: Ohhh ok!

File: packages/vite/src/utils/generator-utils.ts (hunk 2 above)
Comment (mandarini) [reply packages/vite/src/utils/generator-utils.ts:739]: Sure!!

**Pull request merged**